src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -90,6 +87,10 @@
 
       (conj units literal))))
 
+(defn first-child-mention [children]
+    (and (= (count children) 3)
+         (empty? (:literal (first children)))
```

Comment:
I would use the threading operator, to avoid nesting

```
(and (= count children) 3)
     (empty? (-> children first :literal) ) ; this could also be `(empty? (get-in children [0 :literal])`
     (= :mention (-> children :type keyword)))
```

I prefer to have `(= :mention` first, as it's clearer what you want it to be

Also, are you sure about the order?

in your code it's `(:type (keyword (second children)))`, should it not be `(keyword (:type (second children)))` instead?

Reply:
resolved